### ISSUE TITLE
rename v8.13 branch to 2021.02

### ISSUE THREAD

**Issue by gares** [coq/platform#83]
maybe this should be done before the release? @MSoegtropIMC

**Comment by MSoegtropIMC** [comment]
Yes, I thought about this as well. I can do it later today. Please feel free to do it right away.

**Comment by Zimmi48** [comment]
FTR, GitHub now supports branch renaming (see the edit icon on https://github.com/coq/platform/branches). We should do the renaming using this feature because it will retarget existing PR and show a message to users that they need to update their git repository.

Also, it might be worth cleaning up the master README a bit to look more like a homepage and less like a scary development thing (another solution is to change the default branch of the GitHub repository to correspond to the latest stable release instead of being the `master` branch).

**Comment by Zimmi48** [comment]
Yet another solution is to move most of the contents of the current README to the releases section in the GitHub repo. Then, the README would be a more static thing that gives the general objective and points to the releases section. If you want I can do a test with existing releases so that you can see how this would look like.

**Comment by MSoegtropIMC** [comment]
Thanks for the pointers!

An issue with changing the default branch is that CI would need rework to allow nightly builds on master, although in the end I also want nightly builds on the release branches (since stuff like Cygwin can break).

People coming in via the release page would see the ReadMe of the tag.

**Comment by MSoegtropIMC** [comment]
How about splitting the master branch ReadMe into a "User section" pointing to the release branches and a "Devlopper" section with the current information.

**Comment by Zimmi48** [comment]
> How about splitting the master branch ReadMe into a "User section" pointing to the release branches and a "Devlopper" section with the current information.

Sounds pretty good to me.

>People coming in via the release page would see the ReadMe of the tag.

This is a good point, but so far we have advertised the home page of the repository rather than the release page.

**Comment by MSoegtropIMC** [comment]
I did the following:

- renamed the release branch
- bumped the version to 2021.02.0 and updated branch references in the readme files
- slightly cleaned up the README files (and used Enrico's wording for the initial lines)
- reworked the README file on the master branch as discussed
- merged the snap package readme into the top level README_Linux.md file
- (this morning) removed all opam packages already merged upstream (on the release branch and master, but on master I missed one).

The master branch stuff is already merged, for the release branch there is a PR open. I will wait for the CI to finish, merge it and then tag the Release.

On the master branch I only updated the main README file. The platform README files are still in the old state - I will merge this over tomorrow, but only developer docs point to these.

I keep this issue open for further notes until the release is done.
